test/python/circuit/library/test_pauli_feature_map.py

```diff
@@ -128,6 +128,93 @@ def test_zzfeaturemap_raises_if_too_small(self):
         with self.assertRaises(ValueError):
             _ = ZZFeatureMap(1)
 
+    def test_parameter_prefix(self):
+        """Test the Parameter prefix"""
+        encoding_pauli = PauliFeatureMap(
+            feature_dimension=2, reps=2, paulis=["ZY"], parameter_prefix="p"
+        )
+        encoding_z = ZFeatureMap(feature_dimension=2, reps=2, parameter_prefix="q")
+        encoding_zz = ZZFeatureMap(feature_dimension=2, reps=2, parameter_prefix="r")
+        x = ParameterVector("x", 2)
+        y = Parameter("y")
+
+        self.assertEqual(
+            str(encoding_pauli.parameters),
+            "ParameterView([ParameterVectorElement(p[0]), ParameterVectorElement(p[1])])",
+        )
+        self.assertEqual(
+            str(encoding_z.parameters),
+            "ParameterView([ParameterVectorElement(q[0]), ParameterVectorElement(q[1])])",
+        )
+        self.assertEqual(
+            str(encoding_zz.parameters),
+            "ParameterView([ParameterVectorElement(r[0]), ParameterVectorElement(r[1])])",
+        )
+
+        self.assertNotEqual(
+            str(encoding_pauli.parameters),
+            "ParameterView([ParameterVectorElement(x[0]), ParameterVectorElement(x[1])])",
+        )
+        self.assertNotEqual(
+            str(encoding_z.parameters),
+            "ParameterView([ParameterVectorElement(x[0]), ParameterVectorElement(x[1])])",
+        )
+        self.assertNotEqual(
+            str(encoding_zz.parameters),
+            "ParameterView([ParameterVectorElement(x[0]), ParameterVectorElement(x[1])])",
+        )
+
```

Comment:
These are not really necessary, you already checked if the parameters have another value by equality -- so they cannot have another value! 😄 
```suggestion
```